src/logic/safe/store/reducer/__tests__/gatewayTransactions.test.ts

```diff
@@ -236,7 +149,7 @@ describe('gatewayTransactionsReducer', () => {
                 0: [mockNextTx.transaction],
               },
               queued: {
-                1: [mockQueuedTx.transaction],
+                1: [mockQueuedTx.transaction, mockQueuedTx.transaction],
```

Comment:
These are duplicates. In theory the CGW shouldn't return two of the same transaction. There should only be one here.